src/app/pages/settings/containers/settings-page.component.ts

```diff
@@ -223,7 +223,7 @@ export class SettingsPageComponent {
           this.notificationService.sendTestFCMNotification().then(() => {
             this.firebaseAnalytics.logEvent('notification_test', {})
             // NOTE: iOS does not support exitApp()
-            this.platform.exitApp()
+            if (this.platform.is('android')) this.platform.exitApp()
```

Comment:
We exit the app after sending a test notification? what happens on ios in this case?

Reply:
IOS does not support the `exitApp()` function, so it will not do anything here. I added this `exitApp()` because the previous FCM plugin does not show notifications in the foreground so the app would have to be exited to display the notification. The new firebase plugin shows notifications in the foreground, so we can remove this, but I left it there for consistency in case participants get confused (we would have to inform them first of the change).